lib/msal-node/src/cache/Storage.ts

```diff
@@ -166,37 +185,43 @@ export class Storage extends CacheManager {
      * @param inMemory
      */
     removeItem(key: string, type?: string): boolean {
+        this.logger.verbose("removeItem called");
```

Comment:
You can add a `verbosePii` that logs the key as well.

Reply:
Done